src/Evolution/Executables/GeneralizedHarmonic/GeneralizedHarmonicBase.hpp

```diff
@@ -141,6 +141,29 @@ class CProxy_GlobalCache;
 template <bool UseDampedHarmonicRollon, typename EvolutionMetavarsDerived>
 struct GeneralizedHarmonicTemplateBase;
 
+namespace detail {
+template <typename InitialData>
+constexpr auto make_default_phase_order() {
+  if constexpr (evolution::is_numeric_initial_data_v<InitialData>) {
+    return std::array<Parallel::Phase, 8>{
```

Comment:
Just a note, not a request: with C++17 you don't need to specify the template parameters for `std::array`. You can just do `std::array{Parallel::Phase::Initialization, ...}` (note only one curly brace)